packages/react/src/components/Alert/Alert.tsx

```diff
@@ -55,7 +55,7 @@ export interface AlertProps
   /** An alert may contain a header. */
   header?: ShorthandValue<TextProps>
 
-  /** Controls Alert's relation to neighboring items. */
+  /** An alert's position relative to neighboring items. */
```

Comment:
We should standardize on capitalization conventions across all components, but for now I think it's sufficient to make them at least consistent within a single component.